.github/scripts/test.nims

```diff
@@ -9,15 +9,19 @@ let
   rootDir = absolutePath normalizedPath "GITHUB_WORKSPACE".getEnv "."
 
 proc installDeps: void =
+  # debatable if we should fail if unable to install deps
```

Comment:
might as well catch install errors earlier

Reply:
we'll also need to update bookOfNim

Reply:
installing test deps should also check if valgrind already exists and be in a when clause